gptel.el

```diff
@@ -487,7 +487,7 @@ there."
   (let ((prompts-plist
          `(:model ,gptel-model
            :messages [,@prompts]
-           :stream ,(and gptel-stream gptel-use-curl))))
+           :stream ,(or (and gptel-stream gptel-use-curl) :json-false))))
```

Comment:
Thanks, I was fuzzy about the difference between `nil` and `:json-false`.

Could you make just this change the first commit?